docs-site/docs/intro.md

```diff
@@ -2,4 +2,26 @@
 sidebar_position: 1
 ---
 
-# Intentionally Blank
\ No newline at end of file
+# Getting Started
+
+## Blobs
+
+The Blobs project exposes the **data availability** capabilities of [Polkadot](https://polkadot.network) and [Kusama](https://kusama.network) for general use. Use-cases include rollups and inscriptions.
```

Comment:
> inscriptions

this is going to age quick

Reply:
We'll leave it for now, but yeah, it will age quick :)